---
layout: page
title: Biography
permalink: /Bio/
---

<div class="bio-intro">
  <div class="bio-photo">
    <img src="{{ site.baseurl }}{{ site.avatar }}" alt="Declan Bennett" />
  </div>
  <div class="bio-summary">
    <p>I am a computational biologist at <strong>St. Jude Children's Research Hospital</strong> in Memphis, Tennessee. My research focuses on understanding somatic mutations, cancer genomics, and applying computational methods to advance personalized medicine.</p>
  </div>
</div>

## Current Position

In January 2024, I started as a postdoctoral researcher in Dr. Paul Geeleher's lab at St. Jude Children's Hospital. My work primarily focuses on dependency mapping of pediatric cancer models. We're addressing key questions such as which cancer subtypes are poorly represented as cancer cell lines.

## Education & Career

### PhD in Computational Biology
**University of Galway** | Prof. Cathal Seoighe's Research Group

My PhD project aimed to infer the underlying effects of somatic mutations in relatively healthy populations such as the UK Biobank. To answer this research question, I developed analysis pipelines to download, filter, and compute information on somatic mutations in 200,000 whole exome sequenced samples (over 175TB of raw input data!).

### Earlier Experience

I completed my undergraduate degree in **Biotechnology (2016)** and **MSc in Biomedical Genomics (2017)**, both at the University of Galway.

During my third year of undergrad, I spent 3 months in Vannes, Brittany, characterizing the biochemical makeup of *Solieria chordalis*, a seaweed that washes ashore on the beaches of northwest France. For my undergraduate final year project, I worked on finding variants associated with memory formation and schizophrenia under the supervision of Derek Morris.

Coming from a "wet lab" undergraduate degree, my final year project taught me that biological research was not solely pipettes and tedious protocols but that computers and statistics could create insights quickly and easily (and too often from a coffee shop).

My Master's thesis involved using simplistic methods to infer somatic mutation rates from population sequencing data and the germline variants that impact the inferred somatic mutation rate. Throughout my MSc, I built a strong knowledge of cancer biology and statistics while developing a strong computational skillset.

### EMBL-EBI

After graduating from my MSc (2018), I worked as a Bioinformatician at EMBL-EBI for the AMP-T2D project. My work mainly consisted of writing a Type II Diabetes GWAS pipeline to mirror the pipelines at the Broad Institute and push the association statistics to the federated knowledge base at EMBL-EBI. The federated node at the EBI allowed for European genetic data to be housed within Europe while summary-level data could be shared to the knowledge portal hosted at the Broad Institute ([AMP-T2D Knowledge Portal](https://t2d.hugeamp.org/){:target="_blank"}).

### PhD Research at Galway

I returned to Galway to undertake an Irish Research Council-funded PhD scholarship working on somatic mutations from population sequencing projects. In 2020, I moved from the IRC post to a Science Foundation Ireland PhD scholarship within the same research group.

During my PhD, I acted as a de facto system administrator for two high-performance compute clusters. In 2019, I was awarded a **â‚¬25,000 small research equipment grant** to contribute to a new high-performance filesystem (250TB usable storage).

In the first year of my PhD, I was a tutor for probabilistic modeling for molecular biology, covering topics such as Markov chain Monte Carlo methods and models of DNA evolution (Jukes & Cantor, Kimura's 2-parameter model, General Time Reversible, etc.).

After completing my PhD, I undertook a short postdoctoral position at the University of Galway, investigating novel sources of technical errors in NGS data that arose during analysis of the UK Biobank exome data.
